component/main.jsonnet

```diff
@@ -8,12 +8,37 @@ local params = inv.parameters.openshift4_slos;
 
 // Define outputs below
 local mergeSpec = function(name, spec)
-  local slothRendered = std.parseJson(kap.yaml_load('sloth-output/%s.yaml' % name));
+  local slothRendered = std.parseJson(kap.yaml_load('%s/sloth-output/%s.yaml' % [ inv.parameters._base_directory, name ]));
```

Comment:
Is this changed intentionally again?

Reply:
Yes. Otherwise Kapitan tries to load from some `/tmp/XXX` directory.